packages/neuron-ui/src/components/DetectDuplicateWalletDialog/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useMemo, useState, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import Dialog from 'widgets/Dialog'
+import RadioGroup from 'widgets/RadioGroup'
+import { useState as useGlobalState, useDispatch, AppActions } from 'states'
+import { requestPassword } from 'services/remote'
+import styles from './detectDuplicateWalletDialog.module.scss'
+
+const DetectDuplicateWalletDialog = ({ onClose }: { onClose: () => void }) => {
+  const {
+    wallet: { id: currentID = '' },
+    settings: { wallets = [] },
+  } = useGlobalState()
+  const dispatch = useDispatch()
+  const [duplicatedWallets, setDuplicatedWallets] = useState<string[]>([])
+  const [t] = useTranslation()
+
+  const groups = useMemo(() => {
+    const obj: {
+      [key: string]: State.WalletIdentity[]
+    } = {}
+    wallets.forEach(item => {
+      if (item.extendedKey in obj) {
+        obj[item.extendedKey].push(item)
+      } else {
+        obj[item.extendedKey] = [item]
+      }
+    })
+
+    return Object.values(obj).filter(list => list.length > 1)
+  }, [wallets])
+
+  const handleGroupChange = useCallback(
+    (checked: string) => {
+      const [extendedKey, id] = (checked as string).split('_')
```

Comment:
```suggestion
      const [extendedKey, id] = checked.split('_')
```

the `as string` definition here is redundant since `checked: string` is already defined in the arguments